components/header/navbar/searchpage/index.tsx

```diff
@@ -173,8 +178,15 @@ const SearchOverlay = ({
                           Vill du söka filer?{' '}
                           <span
                             onClick={() => {
-                              setIsFetched(false)
-                              setFileSearch(true)
+                              dispatch({
+                                type: 'SET_IS_FETCHED',
+                                payload: false,
+                              })
+                              dispatch({
+                                type: 'SET_FILE_SEARCH',
+                                payload: true,
+                              })
+                              handleSearch(query, true)
```

Comment:
This should now be a single action `SET_FILE_SEARCH`, and the reducer does the necessary changes inside the action handler.